src/code/main.c

```diff
@@ -25,21 +25,26 @@ AudioMgr gAudioMgr;
 OSMesgQueue sSerialEventQueue;
 OSMesg sSerialMsgBuf[1];
 
+#ifdef OOT_DEBUG
 void Main_LogSystemHeap(void) {
     PRINTF(VT_FGCOL(GREEN));
     // "System heap size% 08x (% dKB) Start address% 08x"
     PRINTF("システムヒープサイズ %08x(%dKB) 開始アドレス %08x\n", gSystemHeapSize, gSystemHeapSize / 1024, gSystemHeap);
     PRINTF(VT_RST);
 }
+#endif
 
+// Note: retail_progress says ok but diff score is 100
```

Comment:
in general i wouldnt really too much about diff scores. you can remove the comment